plugins/inventory/foreman.py

```diff
@@ -85,6 +85,60 @@
         description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
         type: integer
         default: 250
+      foreman:
+        use_reports_api:
+            description: Use Reporting API
+            type: boolean
+            default: False
+      report:
+        poll_interval:
+            description: want_organization
```

Comment:
the descriptions of the `report` options might need a bit more work ;)

Reply:
Pushed some changes to add better description. :)